packages/autocomplete/src/index.vue

```diff
@@ -0,0 +1,319 @@
+<template>
+  <div
+    v-clickoutside="close"
+    class="el-autocomplete"
+    role="combobox"
+    aria-haspopup="listbox"
+    :aria-expanded="suggestionVisible"
+    :aria-owns="id"
+  >
+    <el-input
+      ref="inputRef"
+      v-bind="$attrs"
+      :model-value="modelValue"
+      @input="handleInput"
+      @change="handleChange"
+      @focus="handleFocus"
+      @blur="handleBlur"
+      @clear="handleClear"
+      @keydown.up.prevent="highlight(highlightedIndex - 1)"
+      @keydown.down.prevent="highlight(highlightedIndex + 1)"
+      @keydown.enter.prevent="handleKeyEnter"
+      @keydown.tab.prevent="close"
+    >
+      <template v-if="$slots.prepend" #prepend>
```

Comment:
Do we need template v-if for checking solely slots?

Reply:
I'm sure we need. Otherwise, `el-input` will render `prepend/suffix`..